README.md

```diff
@@ -69,9 +69,30 @@ which define throttling behavior for Matrix master run and configuration runs.
 
 Starting from `throttle-concurrents-2.0` the plugin allows throttling particular Pipeline blocks by categories.
 For this purpose you can use the `throttle()` step.
-Throttling within a single job **is not supported**, use features provided by the `parallel()` step or define a special global category for the job.
+Throttling within a single job **is not supported**, you would need to define a special global category for the job.
```

Comment:
:bug: Throttling within a single job *is* supported. You do need a category, but that's orthogonal from whether you can throttle just branches in an individual job.